### ISSUE TITLE
Add endpoint for missing label detection

### ISSUE THREAD

**Issue by lalewis1** [RDFLib/prez#219]
Add an endpoint that returns a list of the subjects and or properties that are not labeled.

Re-use the logic from labelify.

**Comment by recalcitrantsupplant** [comment]
I don't think this is a good idea; it amounts to a SELECT * on the backend. There's no benefit to putting this in Prez over running the equivalent SPARQL query - Prez can't fix the missing labels. What we can do is log URIs that a label is not found for as and when the API "discovers" them. I believe these are already cached with `None` in place of a label, so we could have an endpoint that dumps these ones out. Thoughts on this approach?

**Comment by lalewis1** [comment]
yeah I agree, I think that we could probably live without this feature. Just run it as a sparql query if and when needed.

the alternate implementation you suggest sounds reasonable, but Idk if the extra complexity in Prez is justified, especially with tools like prezmanifest coming along, that should reduce the need to check for labels in Prez anyway.

I could probably just close this ticket then.

**Comment by recalcitrantsupplant** [comment]
Sounds good - I would rather the effort is spent improving the ETL tooling.
